css-four-column-layout/styles.css

```diff
@@ -0,0 +1,26 @@
+body {
+  background-color: rgb(211 211 211 / 64.6%);
+  margin: auto;
+  text-align: center;
+  font-family: Barlow, sans-serif;
+  font-size: 20px;
+}
+
+h1 {
+  text-align: center;
+  font-size: 40px;
+}
+
+.four-columns {
+  flex-basis: 25%;
+}
+
```

Comment:
You should have another class / selector for your phone `img`. Right now, they are taking up so much space they won't fit on my screen. So you need to give your phone `img` a `width: 100%` so they only take up as much space as it's parent